includes/Classifai/Providers/Azure/ComputerVision.php

```diff
@@ -60,7 +60,23 @@ public function register() {
 
 
 	/**
-	 * Process the image via Computer Vision based on the settings.
+	 * Provides the max filesize for the ComputerVision service.
+	 *
+	 * @return int
+	 *
+	 * @since 1.4.0
+	 */
+	public function get_max_filesize() {
+		/**
+		 * Filters the ComputerVision maximum allowed filesize.
+		 *
+		 * @param int Default 4MB.
+		 */
+		return apply_filters( 'classifai_computervision_max_filesize', 4000000 ); // 4MB default.
```

Comment:
WordPress has some fun built-in constants for file sizes and time units, like `MB_IN_BYTES`. Would advise using that here for legibility and technical accuracy.